tools/db_bench_tool.cc

```diff
@@ -3642,6 +3659,17 @@ void VerifyDBFromDB(std::string& truth_db_name) {
       options.compaction_filter = new KeepFilter();
       fprintf(stdout, "A noop compaction filter is used\n");
     }
+
+    if (FLAGS_use_existing_keys) {
+      // Only work on single database
+      assert(db_.db != nullptr);
+      Iterator* iter = db_.db->NewIterator(ReadOptions());
```

Comment:
Should we set `total_order_seek` to true here?

Reply:
Good point. This is difficult for me to remember.